build.sbt

```diff
@@ -100,25 +91,25 @@ lazy val root = (project in file("."))
     scalafixOnCompile := true,
     addCommandAlias("runLocal", "run -Dapplication.router=testOnlyDoNotUseInAppConf.Routes")
   )
+  .settings(CodeCoverageSettings.settings *)
 
 lazy val testSettings: Seq[Def.Setting[?]] = Seq(
   scalafmtOnCompile := true,
   scalafixOnCompile := true,
-  fork := false,
+  fork := true,
   unmanagedSourceDirectories += baseDirectory.value / "test-utils"
 )
 
-lazy val itSettings = Defaults.itSettings ++ Seq(
-  unmanagedSourceDirectories := Seq(
-    baseDirectory.value / "it",
-    baseDirectory.value / "test-utils"
-  ),
-  unmanagedResourceDirectories := Seq(
-    baseDirectory.value / "it" / "resources"
-  ),
-  parallelExecution := false,
-  fork := false
-)
+lazy val it = project.in(file("it"))
+  .enablePlugins(PlayScala)
+  .dependsOn(root % "test->test") // the "test->test" allows reusing test code and test dependencies
+  .settings(DefaultBuildSettings.itSettings())
+  .settings(
+    libraryDependencies ++= AppDependencies.test,
+    Test / fork := true,
+    Test / scalafmtOnCompile := true,
```

Comment:
If I remember correctly yes. We can improve here in following PRs in case we misused anything (minor), as testing this repo is taking quite a lot of time if that makes sense.